app/helpers/countries_helper.rb

```diff
@@ -1,19 +1,16 @@
-require "json"
 module CountriesHelper
-  def get_countries
-    require "uri"
-    require "net/http"
+  class APIFootball
+    include HTTParty
+    API_KEY = Rails.application.credentials.api_sports.football[:key]
+    base_uri  "https://v3.football.api-sports.io/"
+    headers   "x-apisports-key" => API_KEY
 
-    url = URI("https://v3.football.api-sports.io/countries")
 
-    https = Net::HTTP.new(url.host, url.port)
-    https.use_ssl = true
+    def countries(options = {})
+      endpoint = "/countries"
```

Comment:
you could make this a constant and `.freeze` its value as it is not likely to change

Reply:
The endpoints? Like perhaps place them in a constanst hash variable, that's frozen?

What do you think of the below:
```ruby
module CountriesHelper
  class APIFootball
    include HTTParty
    API_KEY = Rails.application.credentials.api_sports.football[:key]
    base_uri  "https://v3.football.api-sports.io/"
    headers   "x-apisports-key" => API_KEY
    ENDPOINTS = {
      countries: "/countries"
    }.freeze

    def countries(options = {})
      queries = { query: options }

      self.class.get(ENDPOINTS[:countries], queries)
    end
```

Reply:
I'd say
```
  ENDPOINT = "/countries".freeze

...

  self.class.get(ENDPOINT, queries)

```

Reply:
Cool, that looks good. I'll make the edit.

Reply:
Oh, one thing however. I'll nedd to make API calls to multiple endpoints ("/leagues", "/teams") to accomplish the other tasks. Should I create a constant array or hash object to store and freeze them. 

Or is it still best to save them to individual constant variables like your suggestion above?